CONTRIBUTING.md

```diff
@@ -18,20 +18,15 @@ Before reporting an issue, see the following wiki articles:
 
 If your issue isn't mentioned there:
 
-- Verify that it hasn't already been reported.
 - If not already running the latest version of Neovim, update to it to see if
-  your problem persists.
+  your issue persists.
+- Verify that it hasn't already been reported.
 - If you're experiencing compile or runtime warnings/failures, try searching for
   the error message(s) you received (if any) on [Neovim's issue tracker][github-issues].
-    - For build issues, see
-      [Troubleshooting#build-issues][wiki-troubleshooting-build-issues].
-    - For runtime issues, see
```

Comment:
This bullet and the one above were removed because Troubleshooting is already mentioned above: `Before reporting an issue, see the following wiki articles: ...`